Automattic/themes
# All Theme Variations: fixes Cover Block text align and font-size settings

This change revises the font scale for the Cover Block to work more intuitively with the new paragraph and heading options in the latest version of the Cover Block. 

To test, for each theme variation:

1. On a post or page, create a new block and make sure that the small, normal, large, and huge font-sizes for the cover text are the same in the editor and the frontend when using `<p>` for the text. 
2. Test to make sure  `<h1>-<h6>` font sizes look correct on the front end when using headings for the text.  
3. Lastly, make sure that the Cover Block text is centered by default for both paragraphs and headings. (Caveat: It appears that the block editor sets the `wp-cover-block h2` to alignleft, and that should be likely be fixed in core).

So far only Elegant Business has this fix applied. Once we test that this fix works for Elegant, we’ll move on to the others with the same fix.
